azure-functions-maven-plugin/src/main/resources/Resources.json

```diff
@@ -0,0 +1,65 @@
+{
+  "$schema": "<TBD>",
```

Comment:
So the TBD will be filled with some logical name like "resources". Or will it reference some json schema that is stored in some schema store?